src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -204,6 +208,79 @@ public void setOptional(boolean optional) {
         this.optional = optional;
     }
 
+    /**
+     * Sets accepted TLS protocol for this channel. Note that this should only be called by
+     * the configuration parser or tests. It is public only for that purpose, mutating protocol state
+     * is probably a bad idea.
+     * @param protocol value to set
+     */
+    public void setProtocol(String protocol) {
+        this.protocol = protocol;
+    }
+
+    /**
+     * Sets accepted TLS protocols for this channel. Note that this should only be called by
+     * the configuration parser or tests. It is public only for that purpose, mutating protocl state
+     * is probably a bad idea.
+     * @param accepted_protocols value to set
+     */
+    public void setaccepted_protocols(List<String> accepted_protocols) {
```

Comment:
Why are we using underscores? We use CamelCase convention. Likely typo? :)

Reply:
It's for dealing with snakeyaml.  After digging in a bit deeper for CASSANDRA-16144, I think it will be looking for a method named "get" + `String.capitalize("accepted_protocols")` so it wouldn't find this anyway.  Method name may have been in desperation on an earlier attempt. I'll see if I can improve it.

Reply:
we should try to find if snakeyaml has something like `@JsonProperty("accepted_protocols")` like Jackson does...

Reply:
boo, it looks like there isn't a way, but we do own how yaml is loaded...  sadly Ekaterina's patch wasn't merged, as it would make it trivial to have yaml names != java names 😭